tasks/system_probe.py

```diff
@@ -182,7 +145,7 @@ def test(
         clang_tidy(ctx)
 
     if not skip_object_files and not windows:
-        build_object_files(ctx, bundle_ebpf=bundle_ebpf)
```

Comment:
Similarly in this task - can we remove `bundle_ebpf` from the definition & always include the bundle tag?